pkg/queue/stats.go

```diff
@@ -102,27 +119,36 @@ func NewStats(podName string, channels Channels, startedAt time.Time) *Stats {
 					totalTimeUsed += val
 				}
 
-				var avg float64
+				var avg, avgp float64
 				if totalTimeUsed > 0 {
-					for c, val := range timeOnConcurrency {
-						ratio := float64(val) / float64(totalTimeUsed)
-						avg += float64(c) * ratio
+					f := func(times map[int32]time.Duration) float64 {
```

Comment:
Does it make sense to define `f` `func` outside `NewStats` `func` and give a more readable name?

Reply:
Done.